.ci/pytorch/test.sh

```diff
@@ -278,6 +278,46 @@ else
   DYNAMO_BENCHMARK_FLAGS+=(--device cuda)
 fi
 
+test_perf_for_dashboard() {
+  TEST_REPORTS_DIR=$(pwd)/test/test-reports
+  mkdir -p "$TEST_REPORTS_DIR"
+
+  local suite="$1"
+  shift
+
+  for dtype in amp float32; do
+    # Run accuracy test
+    # All the accuracy tests can be skipped once the CI accuracy checking is stable enough
+    for backend in eager aot_eager; do
+      python "benchmarks/dynamo/$suite.py" \
+          --accuracy --backend "$backend" "$@" \
+          --output "$TEST_REPORTS_DIR/{$backend}_{$suite}_{$dtype}_training_cuda_accuracy.csv"
+    done
+
+    # Run accuracy test for inductor with different configs
+    # --disable-cudagraphs is the default inductor behavior
+    # TODO: update here once cudagraphs is turned on as default
+    backend=inductor
+    python "benchmarks/dynamo/$suite.py" \
+        --accuracy --backend "$backend" --disable-cudagraphs "$@" \
+        --output "$TEST_REPORTS_DIR/{$backend}_no_cudagraphs_{$suite}_{$dtype}_training_cuda_accuracy.csv"
+    python "benchmarks/dynamo/$suite.py" \
+        --accuracy --backend "$backend" "$@" \
+        --output "$TEST_REPORTS_DIR/{$backend}_with_cudagraphs_{$suite}_{$dtype}_training_cuda_accuracy.csv"
```

Comment:
Note: I will update the Rockset query accordingly, so now we have `inductor_no_cudagraphs` and `inductor_with_cudagraphs`